project/ScalafixBuild.scala

```diff
@@ -190,21 +190,21 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
         s
     },
     commands += Command.command("ci-213") { s =>
-      s"++$scala213" ::
+      s"""set ThisBuild/scalaVersion := "$scala213"""" ::
```

Comment:
wasn't the previous `++` syntax working?

Reply:
once you do `++`, I cannot set specific scalaVersion for any project.